src/translator_cz.h

```diff
@@ -126,7 +126,7 @@
  Translator class (by the local maintainer) when the localized
  translator is made up-to-date again.
 */
-class TranslatorCzech : public Translator
+class TranslatorCzech : public TranslatorAdapter_1_9_5
 {
   public:
 
```

Comment:
Why didn't you change this version? Previously you changed some Czech texts as well.

Reply:
Yes, but I just want make minimal changes only.

Can I include changes of the src/translator_cz.h to this request?

Reply:
Yes no problem at all as this is concerning the same topic so well suited here.

Reply:
Done